components/x11/libXfont/mapfile-vers

```diff
@@ -183,13 +183,9 @@ SUNWprivate {
 	pcfWriteFont;
 	pcfError;
 
-	# "src/bitmap/snfstr.h" - needed by xfs
-	SnfSetFormat;
```

Comment:
Perhaps the rosalinux site overrode the default --enable flags at compile time?

libXfont 1.5.0 from X.Org changed the default from --enable-snfformat to --disable-snfformat:

- https://lists.x.org/archives/xorg-announce/2014-July/002460.html
- https://cgit.freedesktop.org/xorg/lib/libXfont/commit/?id=2a3429413df27224ceeddd22500ce43b5431d698
- https://cgit.freedesktop.org/xorg/lib/libXfont/commit/?id=37595cfd4feaf031552d66f96dc6d58686f9c851